### PR TITLE
add ruby 3.0 and 3.1 in verfiy pipeline

### PR DESCRIPTION
**Pull request by skeshari12**
Signed-off-by: skeshari12 <skeshari@msystechnologies.com>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

**Comment by tas50** [comment]
This should really just be done here https://github.com/chef/cheffish/blob/main/.github/workflows/unit_specs.yml#L17

File: .expeditor/config.yml [hunk 1]
```diff
@@ -44,8 +44,3 @@ subscriptions:
     actions:
       - built_in:rollover_changelog
       - built_in:publish_rubygems
-
-pipelines:
-  - verify:
```
Comment (PrajaktaPurohit) [inline .expeditor/config.yml:49]: Any reason to remove the verify pipeline for this?

Comment (skeshari12) [reply .expeditor/config.yml]: As discussed I have added it again

File: .expeditor/run_windows_tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Stop script execution when a non-terminating error occurs
+$ErrorActionPreference = "Stop"
+
+# This will run ruby test on windows platform
+
+Write-Output "--- Bundle install"
+
+bundle config --local path vendor/bundle
```
Comment (PrajaktaPurohit) [inline .expeditor/run_windows_tests.ps1:8]: would we need to check for the exitcode after this line as well?

File: .github/workflows/unit_specs.yml [hunk 3]
```diff
@@ -12,9 +12,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-18.04, windows-2016, macos-10.15]
+        os: [ubuntu-18.04, macos-10.15]
```
Comment (PrajaktaPurohit) [inline .github/workflows/unit_specs.yml:15]: would this not run any unit tests on windows?

**Comment by marcparadise** [comment]
Replaced with https://github.com/chef/cheffish/pull/219

This uses github actions for windows testing instead of splitting testing between expeditor and actions.

It fixes the action setup so that windows tests run, though currently they are failing for reasons that we need to track down. (We got the same failures on this pipeline, so that's probably a good sign)

**Pull request closed without merging**